lms/djangoapps/course_api/views.py

```diff
@@ -158,11 +162,13 @@ class CourseListView(APIView):
             ]
     """
 
-    def get(self, request):
+    pagination_class = NamespacedPageNumberPagination
+    serializer_class = CourseSerializer
+    paginate_by = 20
```

Comment:
There's already a `PAGINATE_BY` setting: http://www.django-rest-framework.org/api-guide/settings/

And which takes precedence?  `paginate_by` or `page_size_query_param`?